### PR TITLE
WIP: code cleanup

### PR DESCRIPTION
**Pull request by natebird**
- Fixed inconsistent indentation
- Fixed Travis-CI configuration to use a distro with postgresql installed
- Removed extra white spacing

### CODE REVIEW

File: Package.swift [hunk 1]
```diff
@@ -3,7 +3,7 @@ import PackageDescription
 let package = Package(
     name: "PostgreSQL",
     dependencies: [
-   		 .Package(url: "https://github.com/qutheory/cpostgresql.git", majorVersion: 0),
+   		 .Package(url: "https://github.com/vapor/cpostgresql.git", majorVersion: 0),
```
Comment (tanner0101) [inline Package.swift]: We need to give this a minor.

Comment (natebird) [reply Package.swift]: Yep. Updated below.

**Comment by natebird** [comment]
Hmm. I might need to adjust the order a little bit. They had the export above the psql command.

**Comment by natebird** [comment]
Grrr. Will contact Travis support again

**Comment by tanner0101** [comment]
Let's skip travis for now. Great work on this!

**Pull request merged**